src/script/mls/MLSConversations.ts

```diff
@@ -45,9 +45,10 @@ type MLSConversationRepository = Pick<
  * @param conversations - all the conversations that the user is part of
  * @param core - the instance of the core
  */
-export function initMLSConversations(conversations: Conversation[], core: Account): Promise<void> {
+export async function initMLSConversations(conversations: Conversation[], core: Account): Promise<void> {
   const mlsConversations = conversations.filter(isMLSConversation);
-  return joinNewConversations(mlsConversations, core);
+  await joinNewConversations(mlsConversations, core);
+  return core.service?.mls?.schedulePeriodicKeyMaterialRenewals(mlsConversations.map(({groupId}) => groupId));
```

Comment:
This is the main change - consumer (webapp) is responsible for initiating the method by providing the list of group ids

Reply:
I would love to see a type guard here or at least a log for the case that core.service?.mls?.schedulePeriodicKeyMaterialRenewals is not defined.